### ISSUE TITLE
Card ease is being reset for some cards

### ISSUE THREAD

**Issue by levischooley** [ankidroid/Anki-Android#9160]
First, thanks to everyone who contributes to AnkiDroid. This app has been a life-changer for me, and I can't imagine life without it.

Sometime late November of 2020 I had a large amount of cards have their ease reset. The problem seems to persist with some of them even until today.

I did not notice the issue until some weeks/months had gone by, and don't remember what might have been the initial reason for the reset. I remember having a computer crash around that time and maybe having some problems with the sync back to the PC and needing to restore the database from a backup. Otherwise, I do almost all of my reviews on my phone using AnkiDroid.

Here is an example of one of the cards with info taken from the Desktop version of Anki (again, I use AnkiDroid for all reviews):
![image](https://user-images.githubusercontent.com/3987655/123697292-587ce580-d822-11eb-8b1c-ffedb5a16a88.png)

Note that the ease for the card is 250%, but the ease on 2021-06-02 was 190% with an answer rating of '2' (hard, which should have lowered the ease). Note the jump from 130% to 235% on 2020-11-26. I manually edited the database to change the ease to 130% on 2021-01-06, but then it reset again on 2021-02-01.

I have dozens, if not hundreds, of cards that follow this pattern, but not all cards were affected. I manage my cards' eases pretty rigorously, so it feels very inconvenient to have them reset. Some cards in particular need to be at 130% so that I am reviewing them frequently.

I've been trying to suss out some kind of explanation or pattern, but can't make heads or tales of it. Nor do I know how to give instructions for reproducibility. Thanks for your consideration!

**Comment by david-allison-1** [comment]
Hi, "Modify due date" used to reset the ease of cards to 250 (still does on AnkiDroid, with a warning).

If it's not that, check your addons. 

Do you have any reason to believe that this is an AnkiDroid problem, rather than an Anki Desktop problem?

https://forums.ankiweb.net/ would be the best place to diagnose otherwise

**Comment by david-allison-1** [comment]
Hi, you were experiencing a bug with Anki Desktop where the default ease was set to 2.5, instead of 250.

An update of Anki Desktop fixed this for all of your cards

See: https://changes.ankiweb.net/#/?id=changes-in-2134

**Comment by david-allison-1** [comment]
I'm going to close this, as I don't believe that there's action for us to take here.

Please feel free to respond though, if there's anything else that we can do.

Regardless, thanks for the report!

**Comment by levischooley** [comment]
@david-allison-1 
That was fast! Thanks for this info. I use 250 as the default for my vocab decks, but had 130 set as the default for my Scripture verse decks, due to the fact that the card content is longer/more complex, so needs more frequent reviews. And it was those cards that were changed. (Though not all of them... which is weird.) So I think that the bulk of the problem does lie with Anki Desktop.

I've updated to the latest version and set the default ease for those decks to 131% (as instructed). It still feels like AnkiDroid may not be updating the ease correctly, but with this info I will monitor it and do a little more troubleshooting if necessary.

Thanks again!

**Comment by david-allison-1** [comment]
Thanks! If there's any issue with AnkiDroid, please do let us know.

**Comment by levischooley** [comment]
OK, just a follow up. I did my reviews today, and sync'd with Anki Desktop, all while watching the cards' ease settings very carefully, and it appears that everything is functioning normally.

However, I think that part of the problem was that I skipped over the warning in AnkiDroid that the card ease would be reset when rescheduling a card. I was using the reschedule feature quite a lot to manually rebalance future reviews after missed days. It seems that I will switch to Anki Desktop to do rebalancing from now on...

But this leads to the question, If Anki Desktop no longer resets the ease when rescheduling, could AnkiDroid be updated as well? This would make life just a little more convenient for me.

Again, thanks for your time and consideration!
